### PR TITLE
fix(api): run rendering process in a Sandbox Environment

### PR DESCRIPTION
**Pull request by fschuch**
Cherry-pick 4a19d1c4498b53bc6cd0a360d5e9715b254c0efd (PR #667) to the main branch.

### CODE REVIEW

File: jobbergate-api/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 This file keeps track of all notable changes to jobbergate-api
 
 ## Unreleased
+- Fixed issue on the Job Template rendering process by running it inside a Jinja Sandbox Environment
```
Comment (julianaklulo) [inline jobbergate-api/CHANGELOG.md:6]: Does this fix have a task id?

File: jobbergate-api/tests/apps/test_services.py [hunk 2]
```diff
@@ -903,6 +903,23 @@ async def test_render__raises_422_on_bad_template(self, make_upload_file, dummy_
         assert "TemplateSyntaxError" in exc_info.value.detail
         assert "Unable to process jinja template filename=file-one.txt" in exc_info.value.detail
 
+    async def test_render__raises_422_on_sandbox_violation(self, make_upload_file, dummy_file_service):
+        """
+        Test that the ``render()`` method raises a 422 error if the template unsecure.
```
Comment (julianaklulo) [inline jobbergate-api/tests/apps/test_services.py:908]:
```suggestion
        Test that the ``render()`` method raises a 422 error if the template is unsecure.
```

**Review by julianaklulo — APPROVED** [review]
Just two small comments, but otherwise looks good to me

File: jobbergate-api/CHANGELOG.md (hunk 1 above)
Comment (fschuch) [reply jobbergate-api/CHANGELOG.md:6]: It was a quick fix, no ticket number.

**Pull request merged**